index.js

```diff
@@ -1,3 +1,3 @@
 module.exports = function () {
-  return false // we've been rescued!
+  return require('false') // we've been rescued!
```

Comment:
This will not return `false`, it will return `function () { return false }`. The usage then becomes `areFerossAndMafintoshStuckInAnElevator()()` :sob: